tests/tool_excimer_script_metadata_test.php

```diff
@@ -193,6 +198,7 @@ public function sampling_limit_provider(): array {
      */
     public function test_get_redactable_param_names(string $config, array $expected) {
         set_config('redact_params', $config, 'tool_excimer');
+        script_metadata::init();
         $this->assertEquals($expected, script_metadata::get_redactable_param_names());
```

Comment:
is it viable in any way to add a unit test around the insert inside an insert or it that too hairy?

Reply:
A problem with testing this is that the database does not get reset after each unit test when using the alternate connection. If this is important to include, then I can spend some time researching how to do this.

Reply:
No I was expecting it to be a weird thing to try and unit test